app/views/RoomView/index.js

```diff
@@ -52,15 +52,12 @@ export default class RoomView extends LoggedView {
 		const rid = navigation.getParam('rid');
 		const t = navigation.getParam('t');
 		const f = navigation.getParam('f');
```

Comment:
Remove this line and https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/738/commits/6e001300dd8e943b20fd0ba4427b9bf128bd231a#diff-31da83bd6996164d4ba4ea85b1e303ceR148